docker-compose.yml

```diff
@@ -60,9 +59,11 @@ services:
     environment:
       DATABASE_URL: 'postgis://postgres:@postgres/lametro'
       SHARED_DB: "True"
-    command: sh -c 'pupa update --rpm=600 lametro people && pupa update --rpm=600 lametro bills window=30 && pupa update --rpm=600 lametro events'
+      OCD_DIVISION_CSV: '/app/lametro_divisions.csv'
+    command: sh -c 'pupa update --rpm=600 lametro people && pupa update --rpm=600 lametro bills window=14 && pupa update --rpm=600 lametro events'
     volumes:
       - ./lametro/secrets.py:/app/lametro/secrets.py
+      - ./data/lametro_divisions.csv:/app/lametro_divisions.csv
```

Comment:
Can you also add this file to version control? It might be omitted due to the .gitignore, so look for how to explicitly exclude a file from a gitignore directive. 👍

Reply:
Done.